stdlib/2.7/io.pyi

```diff
@@ -18,30 +18,30 @@ class IOBase:
     ...
 
 class BytesIO(BinaryIO):
-    def __init__(self, initial_bytes: str = ...) -> None: ...
+    def __init__(self, initial_bytes: bytes = ...) -> None: ...
```

Comment:
Everything in this file honestly looks like an improvement (I may even keep it regardless of what we do here, since bytes would just be an alias for str).